src/CMakeLists.txt

```diff
@@ -107,11 +107,11 @@ set( SUFFIX_LIB_DEFAULT "" )
 set( SUFFIX_BIN_DEFAULT "" )
 
 # Modify the global find property to help us find libraries like Boost in the correct paths for 64-bit
-# Essentially, find_library calls will look for /lib64 instead of /lib; works for windows and linux
+# Essentially, find_library calls will look for /lib64 instead of /lib; works for windows, not on debian
 if( BUILD64 )
 	set_property( GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS TRUE )
 	message( STATUS "64bit build - FIND_LIBRARY_USE_LIB64_PATHS TRUE" )
-	if( NOT APPLE )
+	if( WIN32 )
```

Comment:
No debian linux system like CentOS still use lib64 folder structure that we need to support

Reply:
Ah, so then it's better to explicitly check for debian systems, I suppose